third_party/cpgfunctionEP-0.3.2/third_party/qdt-master/README.md

```diff
@@ -0,0 +1,67 @@
+# qdt
```

Comment:
The `qdt` library, written by a masters student in 2006. Adolfo Munoz wrote the guassian kronrod integration for a paper he published, and graciously made the functions open source. His library enabled me to get rid of the boost function call to `boost::math::quadrature::kronrod`